project/AppDependencies.scala

```diff
@@ -10,15 +10,16 @@ object AppDependencies {
     "uk.gov.hmrc"             %% "govuk-template"           % "5.42.0-play-26",
     "uk.gov.hmrc"             %% "play-ui"                  % "8.2.0-play-26",
     "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.1.0"
-  )
+  ) map(_.withSources())
```

Comment:
What this will give us?

Reply:
Allows us to look into the source code of those libraries

Reply:
Pretty sure its in our other services, it's very useful. Especially when you get a NullPointer or something inside library code. Without this you just see an interface and a message telling you IntelliJ cant find the sources.

Reply:
`sbt updateClassifiers` probably is similar, but good, let's add this to every service. Nice!